packages/nitro-protocol/contracts/interfaces/IForceMove.sol

```diff
@@ -125,7 +125,7 @@ abstract contract IForceMove {
         uint8 numStates,
         uint8[] memory whoSignedWhat,
         Signature[] memory sigs
-    ) public virtual;
+    ) public virtual returns (bytes32);
```

Comment:
This saves a `require`, some `abi.encode` and a `keccak256` on our happy path. It takes a little bit more gas on the slightly-less happy path.

Reply:
This got cherry-picked off this branch and made it's way into `master` #2980 .